packages/wallet-sdk/src/sign/extension/ExtensionSigner.ts

```diff
@@ -48,11 +48,9 @@ export class ExtensionSigner implements Signer {
   }
 
   async handshake() {
-    const accounts = (await this.request({
+    await this.request({
       method: 'eth_requestAccounts',
-    })) as AddressString[];
-    this.updateListener.onAccountsUpdate(accounts);
```

Comment:
redundant. extension should emit this anyway
![image](https://github.com/user-attachments/assets/fbc021f7-2004-439a-8b9a-49cbbf6d9872)

Reply:
don't we need to set the accounts on the signer here so the provider can return them on L66 here
<img width="629" alt="Screenshot 2024-07-12 at 4 00 17 PM" src="https://github.com/user-attachments/assets/4f63ab2d-b6ac-49bc-a692-c1b9979266b4">

Reply:
`this._accounts = accounts` in the handshake, and then add getters for `accounts` and `chain`

Reply:
that return value isn't being used at all.
and we have getter.
@nateReiners

Reply:
oh I see :P then... should we delete L66